dali/test/python/test_operator_gaussian_blur.py

```diff
@@ -336,3 +337,31 @@ def test_fail_gaussian_blur():
           "Kernel window should have odd length, got: \d*\."
     yield check_fail_gaussian_blur, 10, 0.0, 2, (100, 20, 3), "HWC", 3, "gpu", \
           "Even or non-centered windows are not supported yet, got window with even length: [\s\S]* for sample \d*\."
+
+
+def test_per_frame():
+    def window_size(rng):
+        return np.array(2 * rng.randint(1, 15) + 1, dtype=np.int32)
+
+    def per_axis_window_size(rng):
+        return np.array([window_size(rng) for _ in range(2)])
+
+    def sigma(rng):
+        return np.array((rng.random() + 1) * 3., dtype=np.float32)
+
+    def per_axis_sigma(rng):
+        return np.array([sigma(rng) for _ in range(2)])
+
+    video_test_cases = [
+        (fn.gaussian_blur, {}, [("window_size", window_size, True)]),
+        (fn.gaussian_blur, {}, [("window_size", per_axis_window_size, True)]),
+        (fn.gaussian_blur, {}, [("sigma", sigma, True)]),
+        (fn.gaussian_blur, {}, [
+            ("window_size", per_axis_window_size, True),
+            ("sigma", per_axis_sigma, True)]),
+        (fn.gaussian_blur, {}, [
```

Comment:
I would like to see a test case where the dictionary is not empty to test the new test framework.

Reply:
done